test/space/space.go

```diff
@@ -20,16 +20,31 @@ func WithoutTargetCluster() Option {
 	}
 }
 
+func WithLabel(key, value string) Option {
+	return func(space *toolchainv1alpha1.Space) {
+		if space.Labels == nil {
+			space.Labels = map[string]string{}
+		}
+		space.Labels[key] = value
+	}
+}
```

Comment:
it looks like that it's not used anywhere

Reply:
removed in 9e9d8ec